pkg/agent/bakerapi.go

```diff
@@ -23,6 +23,7 @@ func NewAgentBaker() (AgentBaker, error) {
 
 type agentBakerImpl struct{}
 
+//nolint:revive // ctx is not used, but may be in the future
```

Comment:
directive `//nolint:revive // ctx is not used, but may be in the future` is unused for linter "revive" (nolintlint) 

is a bug with golangci https://github.com/golangci/golangci-lint/issues/3228